content/TrueCommand/Clusters.md

```diff
@@ -0,0 +1,260 @@
+---
+title: "Clusters Screen"
+description: "Descriptions of the options contained inside the TrueCommand Clusters menu."
+weight: 60
+geekdocCollapseSection: true
+tags:
+ - scaleclustering
+aliases:
+ - /truecommand/clustering/
+---
+
+{{< toc >}}
+
+TrueCommand 2.2, in conjunction with TrueNAS SCALE, can create and manage clusters, cluster volumes, and cluster volume shares.
+
+{{< hint danger >}}
+Clusters are an experimental feature in TrueCommand 2.2.
+Before using such features, please back up all your data.
+Do not rely on this for critical data.
+{{< /hint >}}
+
+## Clusters
+
+The **Clusters** screen contains all options related to the cluster feature.
+To see this screen, log in to TrueCommand and click the <span class="iconify" data-icon="mdi:server-network"></span> **Clusters** icon in the upper left.
+
+![ClustersScreen](/images/TrueCommand/2.2/ClustersScreen.png "TrueCommand Clusters Screen")
+
+{{< expand "Option descriptions (click to expand)" "v" >}}
+When no clusters are created, there is a single option on this screen:
+
+| Setting | Value | Description |
```

Comment:
remove the Value column to be consistent with the CORE and SCALE documentation. Or we can do that later when we reorg TC content.